shared-libs/search/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "search",
+  "version": "1.0.0",
+  "description": "Our implemenation of search using (C|P)ouchDB views.",
+  "main": "src/search.js",
+  "scripts": {
+    "test": "./node_modules/.bin/mocha ./test"
```

Comment:
`./node_modules/.bin` is already on the path for npm scripts.  This script can be rewritten as simply
```
mocha ./test
```

Reply:
TIL, nice!